packages/collections/persistent/persistent.pony

```diff
@@ -3,5 +3,7 @@
 
 List - A persistent list with functional transformations.
 
-Map - A persistent map based on Bagwell's hash array mapped trie algorithm.
+Map - A persistent map based on the Compressed Hash Array Mapped Prefix-tree
```

Comment:
I didn't notice on the initial commit of this, but, would it make sense to make each of these an item in a markdown list?